src/ProgramBlockEditor.js

```diff
@@ -278,20 +285,24 @@ class ProgramBlockEditor extends React.Component<ProgramBlockEditorProps, Progra
                             onClick={this.handleClickDelete}
                             aria-pressed={this.deleteIsSelected() ? 'true' : 'false'}
                             key='deleteButton'
+                            tabIndex={this.props.actionButtonDisabled ? '-1' : undefined}
                         >
                             <DeleteIcon className='ProgramBlockEditor__editor-action-button-svg'/>
                         </AriaDisablingButton>
                     </div>
                 </Row>
                 <Row className='ProgramBlockEditor__delete-all-button-container'>
                     <Collapse in={this.deleteIsSelected()}>
-                        <Button
+                        <AriaDisablingButton
                             aria-label={this.props.intl.formatMessage({id:'ProgramBlockEditor.deleteAll'})}
-                            className='ProgramBlockEditor__delete-all-button'
+                            className={'ProgramBlockEditor__delete-all-button'}
+                            disabledClassName='ProgramBlockEditor__delete-all--disabled'
+                            disabled={this.state.showConfirmDeleteAll}
```

Comment:
Do we need this `disabled` property?

Reply:
isn't it one of the required property for AriaDisablingButton component?